packaging/language/bower.py

```diff
@@ -161,6 +181,7 @@ def main():
         offline=dict(default='no', type='bool'),
         production=dict(default='no', type='bool'),
         path=dict(required=True),
+        relative_execpath=dict(default=False, required=False),
```

Comment:
I wasn't even aware that arguments have a type. I would agree with your suggestion, but it should be done also on path. On one side, I don't know how to do that, and on the other side I would like this to be patcheable against 1.9, which is the versión we currently use.